prow/tide/blockers/blockers.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 var (
-	branchRE = regexp.MustCompile(`(?im)\bbranch:[^\w-]*([\w-]+)\b`)
+	branchRE = regexp.MustCompile(`(?im)\bbranch:[^\w-]*([\w-.]+)\b`)
```

Comment:
https://wincent.com/wiki/Legal_Git_branch_names
Include `/` too. That should cover most branch names out there.